pkg/machines/libvirt-common.es6

```diff
@@ -603,6 +603,19 @@ export function CHECK_LIBVIRT_STATUS({ serviceName }) {
     };
 }
 
+export function virtBuildXML(args) {
+    const command = `virt-xml --build-xml ${args}`;
+
+    return cockpit.script(command, null, {err: "message", environ: ['LC_ALL=en_US.UTF-8']});
```

Comment:
I know that we already use en_US.UTF-8 in some other places in machines, but please note that this is in no way guaranteed to exist. Normally one should use C.UTF-8 everywhere, except that it doesn't exist on RHEL/CentOS 7 (which is a major headache..)

So this isn't a blocker for this PR as it doesn't make things worse, but this should be fixed at some point. I created issue #10423 about it.

Reply:
I 'll add it in high priority. Thanks for heads up